libraries/AP_HAL_Linux/RCOutput_AioPRU.cpp

```diff
@@ -53,8 +61,16 @@ void RCOutput_AioPRU::init()
    // Reset PRU
    *ctrl = 0;
 
-   // Load firmware
+   // Loading firmware
+   // for ARM64 using loading PRU code in block instead memcpy
```

Comment:
Yeah, the code tells me that already.  We need a comment as to *why* we need to do this word-wise on ARM64.